mods/cnc/sequences/misc.yaml

```diff
@@ -294,152 +231,156 @@ pips:
 	pip-blue:
 		Start: 5
 	groups: pdigits
-		Start: 0
 		Length: *
 
 overlay:
 	build-valid-desert:
-		Start: 0
 	build-valid-snow:
 		Start: 2
 	build-valid-temperat:
-		Start: 0
 	build-valid-winter:
-		Start: 0
 	build-valid-jungle:
-		Start: 0
 	build-invalid:
 		Start: 1
 	target-select:
 		Start: 3
 	target-valid-desert:
-		Start: 0
 	target-valid-snow:
 		Start: 2
 	target-valid-temperat:
-		Start: 0
 	target-valid-winter:
-		Start: 0
 	target-valid-jungle:
-		Start: 0
 	target-invalid:
 		Start: 1
 
-powerdown:
-	disabled: speed
-		Start: 3
```

Comment:
That shouldn't matter, because the order generator is not bound at all if the widget doesn't exist.